integration/hive/src/main/java/org/apache/carbondata/hive/MapredCarbonInputFormat.java

```diff
@@ -158,7 +158,9 @@ private static CarbonTable getCarbonTable(Configuration configuration, String pa
     try {
       splitList = carbonInputFormat.getSplits(jobContext);
     } catch (IOException ex) {
-      if (ex.getMessage().contains("No Index files are present in the table location :")) {
+      LOGGER.error("Unable to get splits: ", ex);
+      if (ex.getMessage().contains("No Index files are present in the table location :") ||
+          ex.getMessage().contains("CarbonData file is not present in the table location")) {
```

Comment:
As i can see from code, `CarbonData file is not present in the table location` is a FIleNotFoundException message, not IO, so this condition will be always false?